src/main/java/ro/hiringsystem/model/enums/UserType.java

```diff
@@ -0,0 +1,22 @@
+package ro.hiringsystem.model.enums;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@Getter
+@AllArgsConstructor
+public enum UserType {
```

Comment:
Same comment as for InterviewerType

Reply:
Mentioned about how I changed that in the above commentary. (didn't notice this dedicated one, sorry!)